package.json

```diff
@@ -57,6 +66,19 @@
     "@types/prettier": "^2.0.0",
     "@types/prismjs": "^1.16.0",
     "babel-loader": "^8.0.6",
+    "cz-conventional-changelog": "3.2.0",
+    "husky": ">=4",
+    "lint-staged": ">=10",
     "webpack-extension-reloader": "^1.1.4"
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "husky": {
```

Comment:
I'd add `commitlint` and the hook:

```
"hooks": {
      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
...
```

the way it is now is letting me commit with any message, and I think it would be great forcing all messages to follow the pattern so we can use git messages to generate releases.

Reply:
Added